docs/topics/which_historical_backend.md

```diff
@@ -0,0 +1,104 @@
+#<a name="top"></a>Which historical backend must I use?
+IoT Platform provides several backends where creating and maintaining historical datasets regarding the context information handled in real time. Among these backends we can find:
+
+* [HDFS](http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html), the [Hadoop](http://hadoop.apache.org/) distributed file system.
+* [MicroStrategy](https://www.microstrategy.com), the BI component of the platform.
+* [CKAN](http://ckan.org/), our Open Data platform.
+* [STH Comet](https://github.com/telefonicaid/IoT-STH), a Short-Term Historic database built on top of [MongoDB](https://www.mongodb.org/).
+
+Given such a wide variety of backends, the next question is: which one suites better to my use case? This will usually depend on these variables:
+
+* **Amount of data**. If you could stimate the amount of data in the short and the mid/long term, that would be one of the most important decission factors.
+* **Privacy concerns**. Maybe your data can be made public as Open Data; or not.
+* **Desired analytics**. The main final purpose for storing data is to analyze it in order to get value-added insights.
+* **Visualization tools**. Very related to the above, graphical tools are always welcome when dealing with large amounts of data.
+
+Additionally, a fifth variable could be wether the persistence backend allows for **static data loading**. This means the backend is not only used for dynamically creating real context datasets, but other sources of static data of your own can be uploaded as well.
+
+Thus, let's analyze all the possibilities from the above variables point of view.
+
+##HDFS
+This is the best choice if you don't want to care about the amount of data, since HDFS is the standar *the facto* when dealing with real Big Data. Based on a distributed cluster of machines, the HDFS storage capacity is the aggregation of all the cluster machines' capacity. Therefore, such a backend usually handles data volumnes in the range of the Terabyte; even more.
+
+Nevertheless, HDFS is just a file system in charge of managing folders and (very large) files. So, don't expect for analysis or visualization tools. This is something that other tools in the Hadoop ecosystem must provide; in the case of the IoT Platofom, we can find the following ones:
+
+* **MapReduce**. The native analysis paradigm in Hadoop. Based on splting the original problem into many smaller problems (map pahse) and then aggregating all the partial results (reduce phase).
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-iot-stack/pull/48/commits/93452e87445a80dee72b0712650c19246234aeaa